modules/bundles/spleen_segmentation/configs/inference.json

```diff
@@ -0,0 +1,112 @@
+{
+    "dataset_dir": "/workspace/data/medical/spleen",
+    "device": "$torch.device('cuda:0' if torch.cuda.is_available() else 'cpu')",
+    "network_def": {
+        "_target_": "UNet",
+        "spatial_dims": 3,
+        "in_channels": 1,
+        "out_channels": 2,
+        "channels": [
+            16,
+            32,
+            64,
+            128,
+            256
+        ],
+        "strides": [
+            2,
+            2,
+            2,
+            2
+        ],
+        "num_res_units": 2,
+        "norm": "batch"
+    },
+    "network": "$@network_def.to(@device)",
+    "preprocessing": {
+        "_target_": "Compose",
+        "transforms": [
+            {
+                "_target_": "LoadImaged",
+                "keys": "image"
+            },
+            {
+                "_target_": "EnsureChannelFirstd",
+                "keys": "image"
+            },
+            {
+                "_target_": "ScaleIntensityRanged",
+                "keys": "image",
+                "a_min": -57,
+                "a_max": 164,
+                "b_min": 0,
+                "b_max": 1,
+                "clip": true
+            },
+            {
+                "_target_": "EnsureTyped",
+                "keys": "image"
+            }
+        ]
+    },
+    "datalist": {
```

Comment:
To help onboard the pytorch users to get started, I believe the example should follow the basic workflow such as https://github.com/Project-MONAI/tutorials/tree/master/3d_segmentation/ignite . So I think we should simplify and remove this datalist section and other utilities that are task/dataset specific

Reply:
Updated in the latest commit.

Thanks.